### ISSUE TITLE
syntheticMixedBank spaces thresholds in the wrong metric, collapsing most of its rubric levels

### ISSUE THREAD

**Issue by DaniyalMlk** [DaniyalMlk/calibrate#29]
The modal-category report added with the category panel found this immediately: on the default synthetic mixed bank, **44 of the 60 rubric items carry a level that is modal nowhere** — a level the model never expects to be anybody's most likely score.

That is not a plausible bank. Collapsed categories do occur in real calibrations and are exactly what an item analyst looks for, but at three quarters of the bank it is the generator, not the world.

## The cause

`syntheticMixedBank` spreads thresholds at `location + (k - (steps - 1) / 2)` — one logit per step, flat, regardless of the item's discrimination. But whether a middle category is modal does not depend on the raw gap. It depends on the gap measured in the item's own metric, which is the gap times `a`. The generator draws discriminations from a lognormal centred on 1.1, so a typical item gets a scaled gap near 1.1, and that is not enough.

Bisecting for the critical value gives a clean answer, and one worth having in the suite: for a graded item, a middle category is modal somewhere exactly when

```
a * (b_k - b_{k-1}) > 2 * ln 2
```

which is 1.3863, and which is invariant in `a` — the raw critical gap scales as `2 ln 2 / a`, so 1.98 logits at a = 0.7 and 0.46 at a = 3.0. The current flat spacing of 1.0 sits below it for every item with a discrimination under about 1.39, which is most of them.

The jitter has the same problem in reverse: `0.25 * normal` is applied in raw logits, so on a highly discriminating item it is large relative to the spacing that item needs and can reverse a pair outright.

## What to do

Space thresholds in the item's own metric — the spacing option should be a scaled quantity, `a * gap`, with a default comfortably above `2 ln 2`, and the jitter should scale with it. The docstring's claim that "a spacing near one logit is what calibrated rubrics tend to look like" should go with it: one logit is what a rubric with `a = 1` looks like, and the sentence quietly generalised from a single case.

The partial credit family keeps its draw order, since a reversed step there is a legitimate item and the point of generating one is that it happens.

## Done when

- The `2 ln 2` threshold is fixed by a test that derives it numerically rather than asserting a remembered constant
- Threshold spacing is expressed in the metric that decides the outcome, and is an option
- The default bank's collapsed-level rate drops to something a real bank could plausibly show
- Callers that depend on the generator are checked rather than assumed unaffected
- Full suite green, `tsc` clean
